### DISCUSSION TITLE
Direct URLs stopped working suddenly

### DISCUSSION THREAD

**Discussion by UmeshKU539** [docsifyjs/docsify#1804] [Q&A]
We have a docsify documentation website and direct URL (full path to source files) to the different pages stopped working suddenly. We are getting a `The requested URL was not found on this server` error message.
However, when we go to main URL and navigating to these pages from side bar will load the website.
Note: We disabled URL hashing due to some project specific reason. So these direct URL works if we add the `#` manually. Is there any way to fix this issue without enabling the URL hashing.
Thanks in advance.

**Comment by Hugoonreplit** [comment]
It's Working for me. If it's still not working report it in [the issues section](https://github.com/docsifyjs/docsify/issues)
